<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowDoctor</class>
 <widget class="QMainWindow" name="MainWindowDoctor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hasta Kayıt Sistemi - DOKTOR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Hasta Kayıt</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_42">
              <property name="text">
               <string>Kimlik No:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_IDcheck">
              <property name="inputMethodHints">
               <set>Qt::ImhDigitsOnly</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_find">
              <property name="text">
               <string>ARA</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="1">
               <widget class="QGroupBox" name="groupBox_6">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="title">
                 <string>Muayane Bilgileri</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_18">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_19">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_18">
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_20">
                       <item>
                        <widget class="QLabel" name="label_29">
                         <property name="text">
                          <string>Kayıt Tarihi:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_30">
                         <property name="text">
                          <string>Doktor Adı:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_31">
                         <property name="text">
                          <string>Klinik/Birim:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_32">
                         <property name="text">
                          <string>Nabız:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_33">
                         <property name="text">
                          <string>Vücut Sıcaklığı:</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_21">
                       <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_19">
                         <item>
                          <widget class="QLineEdit" name="lineEdit_record_date"/>
                         </item>
                         <item>
                          <spacer name="horizontalSpacer_8">
                           <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                           </property>
                           <property name="sizeHint" stdset="0">
                            <size>
                             <width>21</width>
                             <height>17</height>
                            </size>
                           </property>
                          </spacer>
                         </item>
                        </layout>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit_clinic_2"/>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit_doctorName_2"/>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit_pulse_2"/>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit_Body_Temperature_2"/>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <spacer name="verticalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>13</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_22">
                     <item>
                      <widget class="QLabel" name="label_34">
                       <property name="text">
                        <string>Hasta Şikayeti:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="lineEdit_complaint_2"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QGroupBox" name="groupBox_5">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="title">
                 <string>Genel Bilgiler</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_16">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_11">
                     <item>
                      <widget class="QLabel" name="label_21">
                       <property name="text">
                        <string>Kan Grubu:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_22">
                       <property name="text">
                        <string>Boy:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_23">
                       <property name="text">
                        <string>Kilo:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_24">
                       <property name="text">
                        <string>Özgeçmiş:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_27">
                       <property name="text">
                        <string>Soygeçmiş:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_28">
                       <property name="text">
                        <string>Kullandığı İlaçlar:</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_17">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_15">
                       <item>
                        <widget class="QLineEdit" name="lineEdit_blood_group_2"/>
                       </item>
                       <item>
                        <spacer name="horizontalSpacer_4">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_16">
                       <item>
                        <widget class="QLineEdit" name="lineEdit_height_2"/>
                       </item>
                       <item>
                        <spacer name="horizontalSpacer_5">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_17">
                       <item>
                        <widget class="QLineEdit" name="lineEdit_weight_2"/>
                       </item>
                       <item>
                        <spacer name="horizontalSpacer_6">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="lineEdit_personelHistory_2"/>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="lineEdit_familyHistory_2"/>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="lineEdit_meds_2"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QGroupBox" name="groupBox_7">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="title">
                 <string>Kişisel Bilgiler</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_23">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_24">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_20">
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_25">
                       <item>
                        <widget class="QLabel" name="label_35">
                         <property name="text">
                          <string>Kimlik No:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_36">
                         <property name="text">
                          <string>Adı Soyadı:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_37">
                         <property name="text">
                          <string>Cinsiyet:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_38">
                         <property name="text">
                          <string>Yaş:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_39">
                         <property name="text">
                          <string>Doğum Tarihi:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_40">
                         <property name="text">
                          <string>Cep Telefonu:</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_26">
                       <item>
                        <widget class="QLineEdit" name="lineEdit_ID_2"/>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit_Name_2"/>
                       </item>
                       <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_21">
                         <item>
                          <widget class="QLineEdit" name="lineEdit_gender"/>
                         </item>
                        </layout>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit_Age_2"/>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit_birth_date"/>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit_phoneNumber_2"/>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_27">
                     <item>
                      <widget class="QLabel" name="label_41">
                       <property name="text">
                        <string>Adres:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="lineEdit_Address_2"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_29">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_28">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Tanı:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_diagnosis"/>
            </item>
            <item>
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>Öneriler:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_suggestions"/>
            </item>
            <item>
             <widget class="QLabel" name="label_44">
              <property name="text">
               <string>Tahlil &amp; Rapor:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_test_report">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>10</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_43">
              <property name="text">
               <string>Reçete:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_receipt"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Temizle</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_Save">
              <property name="text">
               <string>Kaydet</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Muayene</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
